benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1226,6 +1226,8 @@
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseManagementCategory", "UserRole": "citizen", "CRUD": "CRUD"},
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseAccessCategory", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseAccessCategory", "UserRole": "citizen", "CRUD": "CRUD"},
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "CaseAccessCategory", "UserRole": "caseworker-sscs", "CRUD": "CRUD"},
```

Comment:
We have two of these but one with capitals? I believe this will cause some issues doing it like this

Reply:
Apparently the uppercase ones are needed for CCD filtering - which means they've probably implemented it wrong on their end.

Reply:
CCD filtering? what's that?